.github/workflows/main.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI book offensive
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+          php-version:
+            - "8.1"
+    steps:
+      - name: Start PostgreSQL
+        run: |
+            sudo systemctl start postgresql.service
+            pg_isready
+            sudo -u postgres psql -c "create user odm WITH PASSWORD 'odm';"
+            sudo -u postgres psql -c "create database odm OWNER odm;"
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: mbstring, xml, ctype, iconv, intl, pdo_pgsql
+
+      - name: Install dependencies
+        uses: "ramsey/composer-install@v2"
+        with:
+           dependency-versions: "${{ matrix.dependencies }}"
+
+      - name: Install PHPUnit
```

Comment:
As this is a final project and not a library, you can use `bin/phpunit` directly, not need to install the special Symfony package.